.github/workflows/renovate-changesets.yml

```
@@ -0,0 +1,127 @@
+            const { execSync } = require('child_process');
+            
+            const CHANGESET_DIR = '.changeset';
+            const PACKAGE_NAME = 'storybook-addon-source-link';
+            
+            function execCommand(command) {
+              return execSync(command, { encoding: 'utf8' }).trim();
+            }
+            
+            function getChangedChangesetFiles() {
+              const changedFiles = execCommand('git diff --name-only HEAD~1 HEAD').split('\n');
+              return changedFiles.filter(file => 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)